eng/common/templates/steps/send-to-helix.yml

```
@@ -0,0 +1,77 @@
+parameters:
+  HelixSource: 'pr/dotnet-github-anon-kaonashi-bot'     # required
+  HelixType: ̓'tests/default/'                           # required
+  HelixBuild: $(Build.BuildNumber)                      # required
+  HelixTargetQueues: ''                                 # required
+  HelixAccessToken: ''                                  # required
+  HelixPreCommands: ''
+  HelixPostCommands: ''
+  WorkItemDirectory: ''
+  WorkItemCommand: ''
+  CorrelationPayloadDirectory: ''
+  XUnitProjects: ''
+  XUnitTargetFramework: ''
+  XUnitRunnerVersion: ''
+  IncludeDotNetCli: false
+  DotNetCliPackageType: ''
+  DotNetCliVersion: ''
+  EnableXUnitReporter: false
+  WaitForWorkItemCompletion: true
+  condition: succeeded()
+  continueOnError: false
+
+steps:
+  - script: powershell eng/common/tools.ps1
```

I'd prefer just using the powershell task
```suggestion
  - powershell: eng/common/tools.ps1
```

---

okay that's easy